documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -215,7 +215,7 @@ What's a tracked resource? A Tracked Resource is an ARM Resource with "location"
 **How to fix the violation**: Add a PATCH operation that allows at least the update of tags for the tracked resource - if the operation does not exist for the service, this fix requires a service side change. 
 If the resource pointed by the rule is not a tracked resource, this warning may be a false positive, please clarify this with your PR reviewer.
 
-**Impact on generated code**: Generated SDK code will expose the corresponding PATCH operation only if it's present in the specification.
+**Impact on generated code**: Generated SDK code will expose the corresponding PATCH operation only if it's present in the specification. If PATCH operation only supports the update of tags and it's called "Update", there may be a "CreateOrUpdate" (if there's a PUT operation already called this way for update purposes) and an "Update" method in the generated SDK, where "Update" would be just for the update of tags, making it a less desired user experience. Please strongly consider adding all mutable properties to the "Update" operation.
```

Comment:
Can we kindly consider making it a little bit less verbose? something like: 

"If PATCH operation only supports updating tags, then you will Have potentially have two operations in the SDK: "Update" & "CreateOrUpdate", the first updates only Tags while the second allows updating a bigger set of properties which is not the best customer experience"